.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: 🐛 Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: 'bug, untriaged'
+assignees: ''
+---
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**OpenSearch Version**
+Please list the version of OpenSearch being used.
+
+**Dashboards Version**
+Please list the version of OpenSearch Dashboards being used.
```

Comment:
What if I use OUI but don't use OpenSearch Dashboards?

Reply:
How would you feel if i change them to:
* ` Please list the version of OpenSearch, if relevant.`
* `Please list the version of OpenSearch Dashboards, if relevant.`

Reply:
Will revisit this in the next round